content/vi/docs/tasks/tools/install-minikube.md

```diff
@@ -0,0 +1,210 @@
+
+### Cài đặt Minikube sử dụng package
+
+Có các gói *thử nghiệm* cho Minikube có sẵn; bạn có thể tìm thấy các gói Linux (AMD64) từ trang [phát hành](https://github.com/kubernetes/minikube/releases) của Minikube trên Github.
+
+Sử dụng công cụ phân phối Linux package của bạn để cài đặt gói phù hợp.
+
+### Cài đặt Minikube thông qua tải xuống trực tiếp
+
+Nếu bạn không cài đặt qua package, bạn có thể tải xuống bản binary và sử dụng đó.
+
+```shell
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
+  && chmod +x minikube
+```
+
+Đây là một cách dễ dàng để thêm Minikube vào môi trường của bạn:
+
+```shell
+sudo mkdir -p /usr/local/bin/
+sudo install minikube /usr/local/bin/
+```
+
+### Cài đặt Minikube sử dụng Homebrew
+
+Một lựa chọn khác là bạn có thể cài đặt Minikube bằng cách sử dụng Linux [Homebrew](https://docs.brew.sh/Homebrew-on-Linux):
+
+```shell
+brew install minikube
+```
+
+{{% /tab %}}
+{{% tab name="macOS" %}}
+### Cài đặt kubectl
+
+Đảm bảo bạn đã cài đặt kubectl. Bạn có thể cài đặt kubectl theo hướng dẫn sau tại [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos).
+
+### Cài đặt Hypervisor
+
+Nếu bạn chưa cài đặt Hypervisor, hãy cài đặt một trong những phần mềm sau đây:
+
+• [HyperKit](https://github.com/moby/hyperkit)
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+• [VMware Fusion](https://www.vmware.com/products/fusion)
+
+### Cài đặt Minikube
+Cách đơn giản nhất để cài đặt Minikube trên macOS là sử dụng [Homebrew](https://brew.sh):
+
+```shell
+brew install minikube
+```
+
+Bạn cũng có thể cài đặt trên macOS bằng việc tải xuống bản binary:
+
+```shell
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 \
+  && chmod +x minikube
+```
+
+Đây là một cách dễ dàng để thêm Minikube vào môi trường của bạn:
+
+```shell
+sudo mv minikube /usr/local/bin
+```
+
+{{% /tab %}}
+{{% tab name="Windows" %}}
+### Cài đặt kubectl
+
+Đảm bảo bạn đã cài đặt kubectl. Bạn có thể cài đặt kubectl theo hướng dẫn sau tại [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
+
+### Cài đặt Hypervisor
+
+Nếu bạn chưa cài đặt Hypervisor, hãy cài đặt một trong những phần mềm sau đây:
+
+• [Hyper-V](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_install)
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+{{< note >}}
+Hyper-V có thể chạy trên 3 phiên bản khác nhau của Windows 10: Windows 10 Enterprise, Windows 10 Professional, và Windows 10 Education.
+{{< /note >}}
+
+### Cài đặt Minikube sử dụng Chocolatey
+
+Cách đơn giản nhất để cài đặt Minikube trên Windows là sử dụng [Chocolatey](https://chocolatey.org/) (chạy với quyền admin):
+
+```shell
+choco install minikube
+```
+
+Sau khi Minikube hoàn tất việc cài đặt, hãy đóng CLI hiện tại và khởi động lại. Minikube sẽ được thêm vào môi trường của bạn tự động.
```

Comment:
Should be:
"Minikube sẽ được tự động thêm vào môi trường của bạn."

Reply:
I have changed to: "Minikube sẽ được tự động thêm vào biến môi trường path của bạn."